.travis.yml

```diff
@@ -2,10 +2,14 @@ sudo: false
 language: node_js
 node_js:
   - '4.2'
+env:
+  - CODACY_PROJECT_TOKEN=80743790b4c84f8da6c14a8c7caaa05c
```

Comment:
After you merge you're going to want to kill this (and rewrite the git history) and protect the environment variable by [defining it in the repository settings](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings). Also, to trigger analysis by Codacy you'll want to enable the post-commit hook [here](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/settings/integrations).

I would have done these things, but I don't have write access to the repo :).